operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md

```diff
@@ -3,24 +3,72 @@
 The Configuration Framework Service \(CFS\) enables Ansible playbooks to run against both running nodes \(node personalization\) and images prior to boot\(image customization\).
 See [Configuration Management Use Cases](Configuration_Management.md#use-cases) for more information about image customization and when it should be used.
 
-Ideally image customization playbooks should be separate from node personalization playbooks. This reduces the likelihood that Ansible content will run in both modes, and reduces the need for conditional checks in the playbooks.
+## Using the a host group
 
-For situations when both image customization and node personalization need to be part of the same playbook, CFS provides the `cray_cfs_image` variable to distinguish between
-node personalization and image customization. When this variable is set to `true`, it indicates that the CFS session is an image customization and the playbook is targeting an image.
+Ansible is most efficient at skipping tasks when the don't apply to any nodes in the current `hosts` target.  For this reason it's recommended that Ansible plays first setup a host group using the `add_host` module, and then use this to target plays for image customization rather than using the `when` conditional.
```

Comment:
Ansible is most efficient at skipping tasks when _they_ don't apply
Note: You forgot the 'y' in they.

and then use this _group_
Note: Add the word group